conda_smithy/templates/meta.yaml.ci-skel.tmpl

```diff
@@ -0,0 +1,71 @@
+{{ '{%' }} set name = "{{ package_name }}" {{ '%}' }}
+{% raw %}{% set version = environ.get('GIT_DESCRIBE_TAG', 'untagged')|string|replace('-','_') %}
+{% set build_number = (environ.get('GIT_DESCRIBE_NUMBER')|int + 1000)|string %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ environ.get('GIT_DESCRIBE_TAG', '') }}
+
+source:
+  git_url: {{ environ.get('FEEDSTOCK_ROOT', '..') }}
+  version: {{ version }}
+
+build:
+  # Uncomment the following line if the package is pure Python and the recipe
+  # is exactly the same for all platforms. It is okay if the dependencies are
+  # not built for all platforms/versions, although selectors are still not allowed.
+  # See https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python
+  # for more details.
+  # noarch: python
+
+  number: {{ build_number }}
+  string: {{ environ.get('GIT_BUILD_STR', '') }}
```

Comment:
```suggestion
  string: {{ [build_number, environ.get('GIT_DESCRIBE_HASH', '')]|join('_') }}
```
for consistency.

Reply:
Maybe we even want to use `PKG_HASH` here to support build matrices. E.g.,
```suggestion
  string: {{ [build_number, ('h' + PKG_HASH), environ.get('GIT_DESCRIBE_HASH', '')]|join('_') }}
```

Reply:
I don't understand why this is needed. Can you please explain @mbargull?

Reply:
If the `PKG_HASH` is not used, the build matrix will not be created I think because the package build strings are equal.